setup.py

```diff
@@ -1,16 +1,43 @@
-from distutils.core import setup
+"""
+This is the setup module for cyhy-feeds.
+
+Based on:
+
+- https://packaging.python.org/distributing/
+- https://github.com/pypa/sampleproject/blob/master/setup.py
+- https://blog.ionelmc.ro/2014/05/25/python-packaging/#the-structure
+"""
+
+from setuptools import setup
+
+
+def readme():
+    """Read in and return the contents of the project's README.md file."""
+    with open("README.md") as f:
+        return f.read()
+
+
+def package_vars(version_file):
```

Comment:
This isn't currently used.  Is the intent to use it in the future?

Reply:
That is an artifact of partial integration of https://github.com/cisagov/skeleton-python-library (for pre-commit) and can be removed if desired.

Reply:
I'd say remove it unless you intend to add a version to cyhy-feeds.

Reply:
I'll do both. I'll remove it from this PR and then add it back in when I integrate the skeleton and add in the version file so it fits in better with the scope of changes.

Reply:
Good solution!

Reply:
Good solution!